build/configs/rtl8730e/Make.defs

```diff
@@ -283,4 +283,10 @@ endef
 
 define DOWNLOAD
   $(TOPDIR)/../build/configs/common_download.sh $(1) $(2) $(3) $(4) $(5) $(6)
-endef
+endef
+
+define EXTERNAL_DOWNLOAD
+ifeq ($(CONFIG_ARCH_BOARD_HAVE_SECOND_FLASH),y)
+  $(TOPDIR)/../build/configs/external_download.sh $(1) $(2) $(3) $(4) $(5) $(6)
```

Comment:
Should we use different script file for external? Is it better than using the same script for both? I want to know your opinion. (not a request to use a file)

Reply:
I think it will be easier to maintain the external download script separately instead of using the same script and also we donot need to encounter second flash related config while downloading config without external flash

Reply:
Ok for separate script for external. But the flow looks strange.
Currently
1. The makefile calls the internal script with all args and calls the external script with all args again
2. Then, each script parses the arg duplicately, and download necessary
How about below?
1. The makefill call some common download script first with all args
2. The common script parses the arg with partition map
    1. If there is a partition for internal, call internal script with internal partition only
    2. If there is an external partition in the arg, call external one with external partition only

@Tirtha-Samsung @kishore-sn @r-prabu How do you think?

Reply:
@Tirtha-Samsung & @sunghan-chang From my opinion download is common for both & not much changes for external, It will be better to maintain in a single script with configs. 
If we have many different things to do for the external download then maintaining a separate file is good but in this case it dosen't looks so.

Reply:
@Tirtha-Samsung What is the final decision? Except partition parsing, is there any difference between internal and external?